services/motion/builtin/move_request.go

```diff
@@ -20,15 +20,13 @@ import (
 
 // moveRequest is a structure that contains all the information necessary for to make a move call.
 type moveRequest struct {
-	config *motion.MotionConfiguration
-
-	planRequest        *motionplan.PlanRequest
-	kinematicBase      kinematicbase.KinematicBase
-	position, obstacle *replanner
+	config        *motion.MotionConfiguration
+	planRequest   *motionplan.PlanRequest
+	kinematicBase kinematicbase.KinematicBase
 }
 
 // plan creates a plan using the currentInputs of the robot and the moveRequest's planRequest.
-func (mr *moveRequest) plan(ctx context.Context) (motionplan.Plan, error) {
+func (mr *moveRequest) plan(ctx context.Context) ([][]referenceframe.Input, error) {
```

Comment:
why are we changing the type to `[][]referenceframe.Input`? Is there a succinct name for that type?

Reply:
There is not.  Perhaps in the future we will add one and this will change

Reply:
If a plan involves multiple components I think we should either:
1. call this method basePlan rather than plan to make it clear that this only returns a subset of the plan
2. have this method take a parameter of `componetnName` so that the caller can specify the component who's reference frame inputs they are interested in

Otherwise the motionplan.Plan and this method contradict each other.